build/lib/release.sh

```diff
@@ -126,8 +126,13 @@ function kube::release::package_src_tarball() {
 # Package up all of the cross compiled clients. Over time this should grow into
 # a full SDK
 function kube::release::package_client_tarballs() {
-   # Find all of the built client binaries
-  for platform_long in "${LOCAL_OUTPUT_BINPATH}"/*/*; do
+  # Find all of the built client binaries
+  local long_platforms=("${LOCAL_OUTPUT_BINPATH}"/*/*)
+  if [[ -n ${KUBE_BUILD_PLATFORMS-} ]]; then
+    long_platforms=(${KUBE_BUILD_PLATFORMS})
```

Comment:
```SC2206: Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.```

Shellcheck isn't happy with this change. We want splitting, so I think you want to `read -a` or ignore the check here